packages/disputer/index.js

```diff
@@ -99,10 +104,10 @@ async function run(
 
     // The EMP requires approval to transfer the disputer's collateral tokens in order to dispute a liquidation.
     // We'll set this once to the max value and top up whenever the bot's allowance drops below MAX_INT / 2.
-    const currentAllowance = await collateralToken.allowance(accounts[0], empClient.empAddress);
+    const currentAllowance = await collateralToken.methods.allowance(accounts[0], empAddress).call();
     if (toBN(currentAllowance).lt(toBN(MAX_UINT_VAL).div(toBN("2")))) {
       await gasEstimator.update();
-      const collateralApprovalTx = await collateralToken.approve(empClient.empAddress, MAX_UINT_VAL, {
+      const collateralApprovalTx = await collateralToken.methods.approve(empAddress, MAX_UINT_VAL).send({
```

Comment:
I thought so but it looks like the liquidator bot sent these approvals with correct gas estimation when we turned them on last week. See this [tx](https://etherscan.io/tx/0x3130dd25c44382a61f13b076f480fc15631350eae847316154b51b67abad6085). The liquidator bot there was running this code version in it's respective `index.js`

Reply:
Hmmm, do you have any understanding of _why_?

I think this is fine for now, but we should probably standardize how we run web3 calls from the bots to ensure we _ensure_ the gas estimation is done since this has caused a hiccup for users in the past.

Reply:
I'll open an issue.